_drafts/submitting-your-first-pull-request.markdown

```diff
@@ -0,0 +1,205 @@
+---
+layout: post
+title: Submitting Your First Pull Request
+author: dave_paquette
+---
+
```

Comment:
Good idea. Not sure if I will get that in for the CdnDevs version but will for the WD version